package.json

```diff
@@ -19,6 +25,7 @@
         "clean": "rm -rf ./build && rm -rf ./node_modules & rm package-lock.json",
         "server": "webpack-dev-server --config webpack/webpack.dev.js",
         "release": "npm run test && npm run build",
```

Comment:
Looks like these scripts could use a cleanup scrub. One awkwardness I see is that this builds both a dev and release bundle, even though I'd expect `release` to only build the latter. I'm not sure I've ever seen `build:dev` before as dev is usually starting up a dev server, not building anything.

Reply:
Agreed -- scripts are a little janky. I've refactored as follows:
- `server`: runs webpack dev server
- `test`: builds webpack dev bundle and runs unit tests
- `release`: builds webpack prod bundle and builds npm package files